go.mod

```diff
@@ -1,3 +1,5 @@
 module go.etcd.io/bbolt
 
 go 1.12
```

Comment:
Can we bump up to `1.14` as well?

Reply:
Shouldn't be necessary. The Go version in the go.mod is the minimum supported Go version, and this still works fine with Go 1.12.  Changing it is also unrelated to this fix, but you could change it in a different commit later...